src/main/resources/webroot/monitoring.html

```diff
@@ -213,6 +213,6 @@
     {{/each}}
 </script>
 
-<script src="custom-js/monitoring.js"></script>
+<script src="custom-js/monitoring.js?v=1.0.2"></script>
```

Comment:
@barbeau this was to make sure the monitoring.js file will be downloaded since there is a change in that one (passing the startTime back to get the iteration relative to the start time of th euser's session). I can remove if if you wnt to.